pkg/crd/testdata/gen/zoo/zoo_types.go

```diff
@@ -33,6 +33,7 @@ type ZooSpec struct {
 }
 type ZooStatus struct{}
 
+// +kubebuilder:resource:path=zooes
```

Comment:
Without overwriting the plural here the new flect version would generate the plural "zoo"

Reply:
Isn't the plural `zoos`?

Reply:
That is what flect changed. They are now generating zoos instead of zooes.

I could have either changed the test file or pinned the path. I picked pinning the path/plural.

I'll change the test file now instead